pynocular/database_model.py

```diff
@@ -229,10 +96,8 @@ def initialize_table(cls, table_name: str, database_info: DBInfo) -> None:
                 or field.type_.__name__ == "ConstrainedFloatValue"
             ):
                 type = Float
-            elif field.type_.__class__ in (AEnumMeta, EnumMeta):
-                type = SQLEnum(
-                    field.type_, values_callable=lambda obj: [e.value for e in obj]
```

Comment:
we need the `values_callable` part back to store enum values in the db instead of enum names